<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lagWizard</class>
 <widget class="QWizard" name="lagWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>877</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Adjust Scan Sync Angle</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/* CSS Styles used throughout the Avinger code base. Some minor variations may exist. */

/* CSS Styles used throughout the Avinger code base. Some minor variations may exist. */
QWizard#deviceWizard {
	background: rgba( 128,128,128, 255 );
}

QLabel {
 font: 30px &quot;DinPRO-medium&quot;;
 color: white;
}

/* Pushbuttons: */


QPushButton {
    background: rgb( 88, 89, 91 );
    border: 1px solid rgb( 88, 89, 91 );
    color: white;
    font: 12pt &quot;DinPRO-Medium&quot;;
    min-width: 164px;
    max-width: 164px;
    min-height: 42px;
    margin: 1px;
    text-align: center;
    border-radius: 10px;
}

 /* make the default button prominent */
 QPushButton:default {
     background:  rgb( 52, 73, 94 );
     border: 1px solid   rgb( 52, 73, 94 );
	 font: 17px &quot;DinPro-bold&quot;;
 }

QPushButton::hover 
{
    background: rgb(52, 73, 94 );
    border-color: rgb( 166, 166, 166 );
    color: rgb( 214, 214, 214);
    border-width: 1px;
}


QPushButton:disabled 
{
    background:  rgb( 51, 51, 51 ); 
    border: rgb( 51, 51, 51 ); 
 }

/* Dialog boxes: */

QDialog {   
 font: 18px &quot;DinPRO-regular&quot;;
 color: white;
 background-color: #909090;
 border: 1px solid #909090;
 border-radius: 5px;        /* This doesn't work */
}

/* List views and edits boxes: */

QLineEdit, QListView {
    color: #d0d0d0;
	font: 18px DinPro-medium;
    selection-color: white;   
    border: 2px groove white;
    border-radius: 4px;
}

/* ---------- Sliders ------------*/

QSlider::groove:horizontal {
border: 1px solid rgb(  191, 191, 191 );
background: white;
height: 10px;
border-radius: 5px;
}

QSlider::sub-page:horizontal {
background: rgb( 51, 51, 51 );
border: 1px solid rgb( 51, 51, 51 );
height: 10px;
border-radius: 6px;
}

QSlider::add-page:horizontal   /* to the right of the handle */
{
background: rgb(  191, 191, 191  );
border: 1px solid rgb(  191, 191, 191  );
height: 10px;
border-radius: 6px;
}

/* modified from the rest of the UI for the background color */
QSlider::handle:horizontal {
  image: url(:images/handle_normal.png);
  margin: -15px 0;
    background:   rgb( 0,0,0 ); 
}

QSlider::handle:horizontal:hover {
	image: url(:images/handle_rollover.png);
    margin: -15px 0;
    background:   rgb( 0, 0, 0 );
}

QSlider::sub-page:horizontal:disabled {
    background:   rgb(  191, 191, 191  );
    border-color:  rgb(  191, 191, 191  );
    border-radius: 6px;
}

QSlider::add-page:horizontal:disabled {
    background:    rgb(  191, 191, 191  );
    border-color:  rgb(  191, 191, 191  );
    border-radius: 6px;
}

QSlider::handle:horizontal:disabled {
	image: url(:images/handle_disabled.png);
    background:  rgb( 0, 0, 0 );
    border: 1px solid  rgb(  128, 128, 128  );
    border-radius: -15px;
}

/* Check boxes: */

QCheckBox {
	spacing: 5px;
}

QCheckBox::indicator {
	border: #909090;
    background: #909090;
	width: 20px;
	height: 20px;
	margin-top: 4px;
}

QCheckBox::indicator:unchecked {
	image: url(:images/checkBox_normal.png);
}

QCheckBox::indicator:checked {
	image: url(:images/checkBox_active.png);
}

/* Group boxes: */

QGroupBox {
    font: 20px DINPro-medium;
    color: rgba(173, 173, 173, 255);
     background-color: #909090;
     border: 2px solid #909090;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }
 

QGroupBox::title {
      subcontrol-origin: margin;
      subcontrol-position: top left; /* position at the top center */
      padding: 0 3px;
}

/* Spin Boxes: */

QAbstractSpinBox {
	font: 18px DinPRO-medium;
	border: 0;
    border-radius: 5px;
}

QAbstractSpinBox::up-button {
	border: 0;
   background: rgba(233,233,235,255);
   border-top-right-radius: 8px;
   margin-top: -2px;
   margin-bottom: -1px;
   padding-bottom: -5px;
   width: 35px;
}

QAbstractSpinBox::up-arrow  {
  image: url(:/images/arrow_up_normal.png);
}

QAbstractSpinBox::up-arrow:hover {
  image: url(:images/arrow_up_rollover.png);
}

QAbstractSpinBox::down-button {
   border: 0;
   background: rgba(233,233,235,255);
   border-bottom-right-radius: 8px;
   width: 35px;
   padding-top: -5px;
   margin-bottom: -2px;
   margin-top: -1px;
}

QAbstractSpinBox::down-arrow:hover {
  image: url(:/images/arrow_down_rollover.png);
}

QAbstractSpinBox::down-arrow {
  image: url(:/images/arrow_down_normal.png);
}

/* Combo Boxes */

QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 30px;
      border: 0px;
     border-radius: 4px;
   background: qlineargradient(spread:pad, x1:1, y1:0.847, x2:1, y2:0, stop:0 rgba(97,  97, 97, 255), stop:1 rgba(145, 144, 144, 255));
	margin: -2px;
 }

 QComboBox::down-arrow {
     image: url(:images/arrow_down_normal.png);
}

QComboBox::down-arrow:hover {
	image: url(:images/arrow_down_rollover.png);
}

 QComboBox::down-arrow:on { /* shift the arrow when popup is open */
     top: 1px;
     left: 1px;
 }


/****************
 *         v3.0            *
 ****************/
QLabel#titlebarLabel {
	background: rgb( 51, 51, 51 );
}
QLabel#titlebarTextLabel {
    font: 24pt &quot;DINPro-Medium&quot;;
    color: white;
}

QLabel#instructionsLabel
{
    background: rgb( 191, 191, 191 );
}

QLabel#instructionsTextLabel
{
    font: 12pt &quot;DINPro-Medium&quot;;
}
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::CancelButtonOnLeft|QWizard::IndependentPages|QWizard::NoBackButtonOnLastPage|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <property name="subTitleFormat">
   <enum>Qt::AutoText</enum>
  </property>
  <widget class="QWizardPage" name="initialRotationPage">
   <property name="title">
    <string>Adjust Scan Sync: Step 1</string>
   </property>
   <property name="subTitle">
    <string> Please wait...</string>
   </property>
   <widget class="liveView" name="rotationView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="counterRotationPage">
   <property name="title">
    <string>Adjust Scan Sync: Step 2</string>
   </property>
   <property name="subTitle">
    <string> Please wait...</string>
   </property>
   <widget class="liveView" name="counterRotationView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="alignmentPage">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Adjust Scan Sync: Step 3</string>
   </property>
   <property name="subTitle">
    <string>Align the two images to determine the Scan Sync angle using the slider.</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1170</width>
      <height>626</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget {
	background: black;
}</string>
    </property>
    <widget class="gridOverlay" name="leftGrid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="gridOverlay" name="rightGrid">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>0</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>DinPRO-Medium</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #717171;
font: 20px DinPRO-Medium;</string>
        </property>
        <property name="text">
         <string>Angle:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="angleLabel">
        <property name="font">
         <font>
          <family>DinPRO-Medium</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #717171;
font: 20px DinPRO-Medium;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>angleLabel</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>531</width>
       <height>34</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 15px DinPro-Medium;	
color: rgba( 128,128,128, 255 );</string>
        </property>
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="angleSlider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="rotatableLabel" name="rightImage">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="rotatableLabel" name="leftImage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>horizontalLayoutWidget</zorder>
    <zorder>rightImage</zorder>
    <zorder>leftImage</zorder>
    <zorder>rightGrid</zorder>
    <zorder>leftGrid</zorder>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>liveView</class>
   <extends>QGraphicsView</extends>
   <header>liveview.h</header>
  </customwidget>
  <customwidget>
   <class>rotatableLabel</class>
   <extends>QLabel</extends>
   <header>Widgets/rotatableLabel.h</header>
  </customwidget>
  <customwidget>
   <class>gridOverlay</class>
   <extends>QLabel</extends>
   <header>Widgets/gridOverlay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
